specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimusers.json

```diff
@@ -244,12 +249,6 @@
           "204": {
             "description": "The user details were successfully updated."
           },
-          "405": {
-            "description": "Administrator user cannot be modified.",
-            "schema": {
-              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
-            }
-          },
```

Comment:
You can add this information in the description of default error response like [here](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/mysql/resource-manager/Microsoft.DBforMySQL/2017-04-30-preview/mysql.json#L467)

Reply:
fixed

---
In reply to: [147818888](https://github.com/Azure/azure-rest-api-specs/pull/1945#discussion_r147818888) [](ancestors = 147818888)